acceptance-tests/tests/helpers/cmd.js

```diff
@@ -54,11 +54,14 @@ function executeWithInput(processPath, args = [], inputs = [], opts = {}) {
     inputs = [];
   }
 
-  // if (global.config.qa) {
-  //   args.push('--qa');
-  // }
+  if (global.config.qa) {
```

Comment:
Any idea why this was previously commented out?

Reply:
Oh, I just found [this in the readme](https://github.com/HubSpot/hubspot-cli/blob/master/acceptance-tests/README.md#getting-started) and it seems relevant.

> Note that if you are testing against a QA portal, not a PROD one, you'll need to add the --qa flag when running the script. There is still an outstanding issue with this because we attempt to add the --qa flag to all hs commands, however it is not available for all commands.

Reply:
Oh that makes sense.  Good call.  I think I was the one that wrote that commend 😅 I'll recomment it out